ci/check-documentation.py

```diff
@@ -86,6 +88,22 @@ def complain(message):
         # check local link validity
 
         for url in sorted([ref.url for ref in doc_page.external_references()]):
+            if url.startswith(omz_github_url):
+                suggested_reference = url[len(omz_github_url):]
+
+                if suggested_reference.startswith(('blob', 'tree')):
+                    md_path_parent_dir_components = '../' * len(md_path_rel.parent.parts)
+
+                    if url.endswith('.md'):
+                        suggested_reference = blob_tree_components + md_path_parent_dir_components \
+                            + suggested_reference
+                    else:
+                        suggested_reference = omz_reference_prefix + suggested_reference
```

Comment:
1. The (previous) value of `suggested_reference` may contain URL escape sequences, which should be removed with `urllib.request.url2pathname`. You can then use `as_posix` from `Path` to make sure the path uses forward slashes.
2. The suggestion should include the backtick characters around the reference to make it clear to the developer that they should be included.